src/components/types.ts

```diff
@@ -324,6 +324,17 @@ export interface QuestionFormat extends DataFormat {
 
 export interface QuestionProps extends QuestionFormat, ConversationProps {}
 
+export interface PromptsFormat {
+  /** A list of prompt strings for users to select from. These prompts will be displayed as interactive elements in the UI. */
+  prompts: string[]
+  /** An optional className to apply to the prompts container. */
+  className?: string
+}
+
+export interface PromptsProps
+  extends PromptsFormat,
+    Omit<ConversationProps, 'messageId'> {}
+
```

Comment:
Display related properties like `className` etc should not be part of the `format` but component `props`
Also, I think its ok to include `messageId`. The component may not use it right now.

```suggestion
export interface PromptsFormat {
  /** A list of prompt strings for users to select from. These prompts will be displayed as interactive elements in the UI. */
  prompts: string[]
}

export interface PromptsProps
  extends PromptsFormat, ConversationProps {
      /** An optional className to apply to the prompts container. */
  className?: string
    }
    

```

Reply:
I noticed the format is also missing the position but this is referred to in messageSpace

Reply:
We are not passing position to the prompts component, so I only mentioned `position` in the storybook. What do you think?

Reply:
The format dictates what fields are expected from a message of that format on the websocket. Even if the field is not used within the component, it is being used by messageSpace. Adding it here and specifying the expected values would be cleaner when integrating with the library.